salt/reboot/init.sls

```diff
@@ -2,6 +2,9 @@
 # Configuration for the reboot manager
 ##################################################
 
+include:
+  - etcd-proxy
```

Comment:
That was in a rush when releasing the latest beta version. With all these changes in this PR it's safe to bring this back.